### ISSUE TITLE
Cannot scroll on page when Hover Zoom is enabled

### ISSUE THREAD

**Issue by brinkdinges** [extesy/hoverzoom#481]
**ATTENTION!** If you don't follow this template then your issue will be immediately closed.

## Description of the problem
I have noticed this issue on multiple sites recently. All on sites that show a hero image and a "scroll for more info" icon of a mac mouse. Finally decided to investigate and turning off hover zoom solved it.

## Url of the page where the problem occurs (not the image url)
https://eurokoncept.ru/project16.html

## Steps to reproduce the problem (include image url here)
1. Open page
2. Try to scroll. 

## OS version, extension version (Chrome only, Firefox is not supported)
Windows 7, Hover Zoom+ 1.0.138, Chrome 78.0.3904.108

**Comment by brinkdinges** [comment]
Ran into this problem again, on this page:
https://vecomach.nl/

Windows 10, Hover Zoom+ 1.0.138, Chrome Version 80.0.3987.106 (Official Build) (64-bit)

**Comment by brinkdinges** [comment]
Found another page: https://www.mg-products.com/

Windows 10, Hover Zoom+ 1.0.156, Chrome Version 85.0.4183.102 (Official Build) (64-bit)

**Comment by extesy** [comment]
Yes, I was able to reproduce that. Seems to be related to the custom scrollbar implementation which overrides normal mouse events.

**Comment by brinkdinges** [comment]
That's what I figured as well. The real crime here is the custom scroll bar, but that's not something within our control :) I have no real experience with web development, but please point me in the right direction if you need help and if you have an idea on how to fix this.
